jupyter-notebooks/gee-integration/gee-integration.ipynb

```diff
@@ -0,0 +1,324 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Order and Delivery to Google Earth Engine\n",
+    "\n",
+    "In this notebook we're going to cover how to order data using the [Orders API](https://developers.planet.com/apis/orders/) and deliver it to [Google Earth Engine (GEE)](https://earthengine.google.com/) using [Planet’s GEE Delivery integration](https://developers.planet.com/docs/integrations/gee/).\n",
+    "\n",
+    "This example demonstrates how to:\n",
+    "1. Define the data to be ordered\n",
+    "2. Build a cloud delivery configuration object (`delivery_config`), which tells the Orders API where to deliver the data\n",
+    "3. Build an order request to be sent off to the Orders API, `iowa_order`\n",
+    "4. Create the order and have it deliver to your GEE project\n",
+    "\n",
+    "**Prerequsites:** \n",
+    "- An AOI - `iowa_aoi`\n",
+    "- Item ID(s) - `iowa_images`\n",
+    "- A GEE project with EE API enabled - `planet-devrel-dev`\n",
+    "- A pre-existing GEE [ImageCollection](https://developers.google.com/earth-engine/guides/ic_creating) - `gee-integration-testing`\n",
+    "- An account with a download quota. Not sure if you have download quota? Please [get in touch](https://www.planet.com/contact-sales/)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import planet"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define data to be ordered"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The area of interest (AOI) defined as a polygon\n",
+    "iowa_aoi = {\n",
+    "    \"type\":\n",
+    "    \"Polygon\",\n",
+    "    \"coordinates\": [[[-91.198465, 42.893071], [-91.121931, 42.893071],\n",
+    "                     [-91.121931, 42.946205], [-91.198465, 42.946205],\n",
+    "                     [-91.198465, 42.893071]]]\n",
+    "}\n",
+    "\n",
+    "# The item IDs we wish to order\n",
+    "iowa_images = ['20200925_161029_69_2223', '20200925_161027_48_2223']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define cloud delivery location\n",
+    "This is the *key concept* of this notebook. Here, we are defining a cloud delivery configuration object, `cloud_config`, where we are defining the destination to be Google Earth Engine, in the project named `planet-devrel-dev`, in the ImageCollection named `gee-integration-testing`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Google Earth Engine configuration\n",
+    "cloud_config = planet.order_request.google_earth_engine(\n",
+    "    project='planet-devrel-dev', collection='gee-integration-testing')\n",
```

Comment:
If this were a regular script I would 100% do this, but I fear that would be a little inconsistent with how we typically lay out our notebooks. Thoughts on this @mkshah605?

Reply:
I'd be open to it! It might make sense so users can easily change their credentials. It's similar to how I laid out[ this Subscriptions notebook](https://github.com/planetlabs/notebooks/blob/master/jupyter-notebooks/subscriptions_api_tutorial/Subscriptions_API_Quickstart.ipynb), where people can define their bucket and gcs key in cell #2. Definitely open to your thoughts!

Reply:
I've thought about it and we only use the project and collection names a single time, just in this cell, and it feels a bit unnecessary to define these as constants. @mkshah605, it makes sense in your subscriptions notebook, as you use `bucket` and `gcs_key` more than once in your notebook, but in this case I seem to justify creating two more variables. Are you opposed to leaving them as is?

Reply:
yes that makes sense. I am good with leaving them in here